vitrina/datasets/templates/vitrina/datasets/datatable.html

```diff
@@ -85,10 +85,15 @@
                    id="change_resource">{% translate "Redaguoti" %}</a>
             </td>
             <td>
-                <a href="{%url 'resource-delete' pk=r.id %}"
-                   onclick="return confirm('Ar jūs tikrai norite trinti šį šaltinį?')"
-                   class="button is-link is-small is-pulled-right"
-                   id="delete_resource">{% translate "Trinti" %}</a>
+                <form method="post" action="{% url 'resource-delete' pk=r.id %}">
+                    {% csrf_token %}
+                    <button type="submit"
+                            class="button is-link is-small is-pulled-right"
+                            onclick="return confirm('Ar jūs tikrai norite trinti šį šaltinį?')"
```

Comment:
I might be wrong, but I think the word used does not have the definition of "deleting something":

```suggestion
                            onclick="return confirm('Ar jūs tikrai norite ištrinti šį šaltinį?')"
```

What my assumptions are based on:
- https://www.zodynas.lt/terminu-zodynas/t/trinti

Reply:
Tbf, I left this text as it was. But yeah, "ištrinti" is better. Hmm, it would be even better if I also translated this text 🤔

Reply:
Fixed in f5897e7ae6dcade5c131943ebbda3dbead15ebcd